config.toml

```diff
@@ -70,62 +70,62 @@
 ###
 
 
-# max-task-db-size = "100 GiB"
+# max_task_db_size = "100 GiB"
```

Comment:
Would it be possible to check that it works with either a string like that and also a native number (without quotes)?

Reply:
It works with native numbers but I think it’s less readable.
I can either use "100 GiB" or 100000000000, both works fine.

What’s your preference ?